x-pack/plugins/canvas/public/lib/aeroelastic_kibana.js

```diff
@@ -16,7 +17,13 @@ export const aeroelastic = {
   },
 
   createStore(initialState, onChangeCallback = () => {}, page) {
-    if (stores.has(page)) throw new Error('Only a single aeroelastic store per page should exist');
+    if (stores.has(page)) {
+      throw new Error(
+        i18n.translate('xpack.canvas.lib.aeroelasticKibana.createStoreErrorMessage', {
```

Comment:
createStoreErrorMessage => singleStoreErrorMessage

Reply:
This doesn't need translation, it is an internal message.